src/test/java/com/rackspace/salus/telemetry/etcd/services/ZoneStorageTest.java

```diff
@@ -301,4 +304,51 @@ private void revokeLease(long leaseId) {
     client.getLeaseClient().revoke(leaseId).join();
 
   }
+
+  @Test
+  public void testGetExpiredPollerResourceIdsInZone() throws InterruptedException {
```

Comment:
if changing the other
```suggestion
  public void testGetExpiringPollerResourceIdsInZone() throws InterruptedException {
```